src/helm/config/model_deployments.yaml

```diff
@@ -36,10 +33,7 @@ model_deployments:
       args: {}
     window_service_spec:
       class_name: "helm.benchmark.window_services.ai21_window_service.AI21WindowService"
-      args:
```

Comment:
Related to my other comment, I think these changes should be in a different PR since they are functionally different than the other changes.

Reply:
Removed these changes.